guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -37,27 +37,35 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
-To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
-
-* `tfm-rubygem-foreman_discovery`
-ifdef::satellite[]
-* `foreman-discovery-image`
+.Procedure
+. Install Discovery on {ProjectServer}:
++
+ifndef::satellite,orcharhino[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
```

Comment:
```suggestion
--foreman-proxy-plugin-discovery-install-images=true
```

You need install images, but not the URL, you removed the wrong line ;)

Reply:
Addressed :+1:

Reply:
Still not addressed. Please carefully review your changes before saying it is addressed.